source/Wire-Protocol.rst

```diff
@@ -1,5 +1,16 @@
+=============
+Wire Protocol
+=============
+
 This document specifies the makeup of a dripline message and how it is expressed as an AMQP message. 
 
+.. _foundation
+
+Foundation
+==========
+
+Dripline messages are built on the `AMQP 0.9.1 <https://www.rabbitmq.com/protocol.html>`_ messaging protocol as implemented in `RabbitMQ <https://www.rabbitmq.com>`_
+
 
 .. _structure:
 
```

Comment:
(on line 50 or 55 below, sorry can't comment there)
is the type of the `sender_info.versions` supposed to be a dict/table?  or a string?  or variable?

Reply:
it's a dict of dicts.  something like:
```
dripline-cpp:
    version: 2.3.0
    package: dripline-cpp
    commit: oasdfj;al24309r23nla;dlf2
dripline-python:
    version 4.3.0
    package: dripline-python
    commit: f;234r9031rhaf0924hap9348rh
```

Reply:
two comments:
- the package field is duplicate to the key information; is that universally true or just for this example?
- I think it would be helpful to give an example, because it could be interpreted as a list of dicts or dict of dicts
  - for completeness, the example should include the wp version?